autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -280,6 +280,12 @@ variable "monitoring_enable_managed_prometheus" {
   default     = false
 }
 
+variable "monitoring_enabled_components" {
+  type        = list(string)
+  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS (provider version >= 3.89.0). Empty list is default GKE configuration."
```

Comment:
Looks like the underlying module currently requires TPG >=5.9.0, so we can drop the provider version note.

```suggestion
  description = "List of services to monitor: SYSTEM_COMPONENTS, WORKLOADS. Empty list is default GKE configuration."
```

Ideally we would remove from this line as well: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/autogen/main/variables.tf.tmpl#L765

Reply:
Sure! I can update there as well.

Reply:
I pushed the result of that change in a second commit. Do with it as you see fit. :)